action.yaml

```diff
@@ -11,6 +11,8 @@ inputs:
     description: "path to diff file"
   module:
     description: "the Go module name"
+  ignore-main:
+    description: "ignore main package"
```

Comment:
I think you can also add `default: "true"` here since we are ignoring main by default